artwork.html

```diff
@@ -275,200 +359,437 @@ <h2 class="sectionHeading sectionHeading-centered sectionHeading-spaced">Frequen
+                        <p>
+                            <img class="muralModal--icon" src="images/palette.svg">
+                            <span class="muralModal--bold">Lead Artist</span><span class="muralModal--text">: Rachel Wolfe-Goldsmith</span>
+                        </p>
+                        <p>
+                            <img class="muralModal--icon" src="images/map-marker.svg">
+                            <span class="muralModal--bold">Location</span><span class="muralModal--text">: 22nd &amp; Broadway</span>
+                        </p>
+                        <div class="muralModal--end">
+                            <img class="muralModal--icon" src="images/calendar.svg">
+                            <span class="muralModal--bold">Date</span><span class="muralModal--text">: October 2020</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="muralModalContainer" data-mural-name="headup">
+                <div class="muralModal">
+                    <div class="muralModal--img"><img src="images/artwork/headup.jpg"></div>
+                    <div class="muralModal--caption">
+                        <h2 class="muralModal--name">Keep Your Head Up: Woman in Water</h2>
+                        <button class="muralModal--close" aria-label="Close">×</button>
+                        <p>
+                            <img class="muralModal--icon" src="images/palette.svg">
+                            <span class="muralModal--bold">Lead Artist</span><span class="muralModal--text">: Andre Jones</span>
+                        </p>
+                        <p>
+                            <img class="muralModal--icon" src="images/map-marker.svg">
+                            <span class="muralModal--bold">Location</span><span class="muralModal--text">: 19th &amp; Broadway</span>
+                        </p>
+                        <div class="muralModal--end">
+                            <img class="muralModal--icon" src="images/calendar.svg">
+                            <span class="muralModal--bold">Date</span><span class="muralModal--text">: December 2020</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="muralModalContainer" data-mural-name="sph">
+                <div class="muralModal">
+                    <div class="muralModal--img"><img src="images/artwork/sph.jpg"></div>
+                    <div class="muralModal--caption">
+                        <h2 class="muralModal--name">San Pablo Hotel</h2>
+                        <button class="muralModal--close" aria-label="Close">×</button>
+                        <p>
+                            <img class="muralModal--icon" src="images/palette.svg">
+                            <span class="muralModal--bold">Lead Artist</span><span class="muralModal--text">: Derrick "Rtystk" Shavers</span>
```

Comment:
on the spreadsheet (https://docs.google.com/spreadsheets/d/1RXPV8kqsDCZ6gDK1Jp8YM9o-01NhK7qokUNO9favf80/edit#gid=0) it says that Andre is the artist, not Derrick.